src/resources/js/v2/tickets-utils.js

```diff
@@ -159,13 +159,7 @@ tribe.tickets.utils = {};
 			return Math.round( num * k ) / k;
 		};
 
-		let s = ( prec ? toFixedFix( n, prec ) : Math.round( n ) ).toString().split( dec );
-
-		// if period is the thousands_sep we have to spilt using the decimal and not the comma as we work
-		// with numbers using the period as the decimal in JavaScript.
-		if ( '.' === format.thousands_sep ) {
-			s = ( prec ? toFixedFix( n, prec ) : Math.round( n ) ).toString().split( '.' );
-		}
+		let s = ( prec ? toFixedFix( n, prec ) : Math.round( n ) ).toString().split( '.' );
```

Comment:
That seems like it would break thing when using `,` as the separator.

Reply:
The `numberFormat()` method won't be receiving a formatted price, though. It will always be a regular decimal (.) with no thousands separator. This is because we are either reading straight from the `data-ticket-price` attribute, which is a regular float value with a decimal (.), or we're running the price text through the `cleanNumber()` method to convert it to a regular float with a decimal (.).